static/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -277,6 +282,90 @@ export default class SentryApplicationDetails extends AsyncView<Props, State> {
     }
   };
 
+  getAvatarPreview = (avatarStyle: 'color' | 'simple') => {
+    const {app} = this.state;
+    const styleMap = {
+      color: {
+        size: 50,
+        title: t('Default Logo'),
+        description: t('The default icon for integrations'),
+      },
+      simple: {
+        size: 20,
+        title: t('Default Icon'),
+        description: t('This is an optional icon used for Issue Linking'),
+      },
+    };
+    return (
+      app && (
+        <AvatarPreview>
+          <StyledPreviewAvatar
+            size={styleMap[avatarStyle].size}
+            sentryApp={app}
+            isDefault
+          />
+          <AvatarPreviewTitle>{styleMap[avatarStyle].title}</AvatarPreviewTitle>
+          <AvatarPreviewText>{styleMap[avatarStyle].description}</AvatarPreviewText>
+        </AvatarPreview>
+      )
+    );
+  };
+
+  getAvatarModel = (avatarStyle: 'color' | 'simple'): Model => {
+    const {app} = this.state;
+    const defaultModel: Model = {
+      avatar: {
+        avatarType: 'default',
+        avatarUuid: null,
+      },
+    };
+    if (!app) {
+      return defaultModel;
+    }
+    const isColor = avatarStyle === 'color';
+    return {
+      avatar:
+        (app?.avatars || []).find(({color}) => color === isColor) || defaultModel.avatar,
```

Comment:
I think the variable names the filter `color === isColor` are going to get confusing. I would push `avatarStyle` as far  up as possible (i.e. where we get the value from the API). That way you can do 
```tsx
(app?.avatars || []).find(({avatarStyle}) => avatarStyle === 'color') || defaultModel.avatar
```
The reason I'd do this is that one day there might be a third `avatarStyle` value.

Reply:
I think its indicative of it being confusing but the field on the model is `color: boolean`, I tried to be more explicit with my own parameter of `avatarStyle: 'color' | 'simple'`, but it just confused things. I've reverted these `get` methods to accept `isColor: boolean`, same as the DB.